sdkproject/Assets/Mapbox/Unity/Editor/PropertyDrawers/SpawnPrefabOptionsDrawer.cs

```diff
@@ -0,0 +1,41 @@
+namespace Mapbox.Unity.Map
+{
+	using System.Collections;
+	using System.Collections.Generic;
+	using UnityEngine;
+	using Mapbox.Unity.MeshGeneration.Modifiers;
+	using UnityEditor;
+
+	[CustomPropertyDrawer(typeof(SpawnPrefabOptions))]
+	public class SpawnPrefabOptionsDrawer : PropertyDrawer
+	{
+		static float lineHeight = EditorGUIUtility.singleLineHeight;
+
+		private GUIContent prefabContent = new GUIContent
+		{
+			text = "Prefab",
+			tooltip = "The prefab to be spawned"
+		};
+
+		private GUIContent scalePrefabContent = new GUIContent
+		{
+			text = "Scale down with world",
+			tooltip = "Turning on this option causes the prefab to scale down when you scale the map object"

```

Comment:
-> "The prefab will scale with the map object" [keep it short; feel free to tweak this if it's not exactly right]